### PR TITLE
Added outline (Wiki)

### PR DESCRIPTION
**Pull request by tonyxu-io**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``- [Name](http://homepage/) - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

- [x] Your additions are ordered alphabetically.
- [x] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md) and marked `⊘ Proprietary`.
- [x] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x] Your additions have their `License` main server-side `Language`/platform/requirement listed.
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/Kickball/awesome-selfhosted/issues) or [PRs](https://github.com/Kickball/awesome-selfhosted/pulls), incluing closed ones.
- [x] If the main link of your addition points to the program's source code, there is no need to add a duplicate _Source Code_ link. The _Demo_ and _Clients_ links are not mandatory.
- [x] Any category you are creating has the minimum requirement of 3 items. If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1313,6 +1313,7 @@ See also [Documentation Generators](#documentation-generators), [Wikimatrix](htt
 - [Mediawiki](https://www.mediawiki.org/wiki/MediaWiki) - MediaWiki is a free and open-source wiki software package written in PHP. It serves as the platform for Wikipedia and the other Wikimedia projects, used by hundreds of millions of people each month. ([Source Code](https://phabricator.wikimedia.org/diffusion/MW/)) `GPL-2.0` `PHP`
 - [MoinMoin](https://moinmo.in/) - Advanced, easy to use and extensible WikiEngine with a large community of users. ([Source Code](http://hg.moinmo.in/moin)) `GPL-2.0` `Python`
 - [Olelo](https://github.com/minad/olelo) - Olelo is a wiki that stores pages in a Git repository, supports many markup styles and has an extensible, hackable architecture. ([Demo](http://gitwiki.org/)) `MIT` `Ruby`
+- [Outline](https://github.com/outline/outline) `⚠` - An open, extensible, wiki for your team built using React and Node.js. `BSD-3-Clause` `Nodejs`
```
Comment (n8225) [inline README.md]: add https://www.getoutline.com/ as site page and move https://github.com/outline/outline to demo

**Comment by tonyxu-io** [comment]
I don't get it why https://github.com/outline/outline should be `demo`. https://www.getoutline.com/ is better candidate for `demo` I guess?

File: README.md (hunk 1 above)
Comment (n8225) [reply README.md]: My apologies, I typed that wrong.  https://github.com/outline/outline would work as site or demo.
But I think 
Site: https://www.getoutline.com/    Source: https://github.com/outline/outline
would be better

**Comment by tonyxu-io** [comment]
Updated with required changes.

**Comment by tonyxu-io** [comment]
Any update on merging this PR?

**Comment by nodiscc** [comment]
Hi @tonyxu-io I wwas about to merge this PR, but was wondering why you added the `⚠` symbol? Does Outline require 3rd party services to work properly (like external APIs, messaging accounts, auth providers...)

**Comment by tonyxu-io** [comment]
It requires logging in through Slack, or Google(Upcoming version).

**Pull request merged**